### PR TITLE
Feature/exo 4677 support new typeformfeedback

### PR DESCRIPTION
**Pull request by jsujar**
## Problem

Explain the problem you are fixing (add the link to the related issue(s), if any).

## Solution

Explain the solution that has been implemented, and what has been changed.

## Commandments

- [ ] Write PEP8 compliant code.
- [ ] Cover it with tests.
- [ ] Pay attention to backward compatibility, or if it breaks it, explain why.
- [ ] Update documentation (if relevant).

### CODE REVIEW

File: typeform_feedback/tasks/generic_typeform.py [hunk 1]
```diff
@@ -2,12 +2,53 @@
...
-    pass
+    def get_user(self, response):
+        form_response = self._get_form_response(response)
+        hidden_fields = self._get_hidden(form_response)
+        user_id = hidden_fields.get(
+            settings.TYPEFORM_FEEDBACK_WEBHOOK_LABEL_HIDDEN_USER_FIELD,
+        )
+        return get_user_model().objects.get(pk=user_id)
+
+    def get_object(self, response):
```
Comment (tomasgarzon) [inline typeform_feedback/tasks/generic_typeform.py:23]: El método get_object está repetido

File: typeform_feedback/urls.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import url
+
+from .views import GetQuizView
+
+app_name = 'typeform_feedback'
+
+urlpatterns = [
+    url(r'^generic/(?P<quiz_slug>[a-z A-Z _]+)/$', GetQuizView.as_view(), name='get-quiz'),
```
Comment (tomasgarzon) [inline typeform_feedback/urls.py:8]: podemos usar la forma de django 2, con el path ?

**Review by tomasgarzon — CHANGES_REQUESTED** [review]

File: typeform_feedback/tasks/generic_typeform.py (hunk 1 above)
Comment (jsujar) [reply typeform_feedback/tasks/generic_typeform.py:23]: Fixed

File: typeform_feedback/urls.py (hunk 2 above)
Comment (jsujar) [reply typeform_feedback/urls.py:8]: Si queremos mantener compatiblidad con Django 1.x deberiamos dejarlo así.

**Pull request closed without merging**